### PR TITLE
♻️ Pincerizing the bot!

### PR DESCRIPTION
**Pull request by Sigmanificient**
Hi there! I successfully transformed your bot to work within the `pincer` library.
This will make a great example to learn you how to switch bot library. Make sure to understand the changes that i done before merging.

If you wanna try the bot with early `0.10.0`, you'll need to use 
```sh
pip install git+https://github.com/Pincer-org/pincer
```

### CODE REVIEW

File: app/cogs/player.py [hunk 1]
```diff
@@ -71,66 +67,55 @@ def trans_paste(fg_img, bg_img, alpha=1.0, box=(0, 0)):
...
+                         if chest.get("index") != 0
+                         else "next")
+                        + f' : {chest.get("name")}'
+                )
+                for chest in chests.get('items')
+            ),
+            inline=False,
+        ).set_image(url="attachment://image0.png")
+
+        return Message(embeds=[info_embed], attachments=[deck])
```
Comment (drawbu) [inline app/cogs/player.py:118]: it needs to return the Embed object directly, not a variable

Comment (drawbu) [reply app/cogs/player.py:118]: we also need to send a first message to avoid Discord shutdown the interaction

**Review by drawbu — CHANGES_REQUESTED** [review]

File: app/cogs/player.py [hunk 2]
```diff
@@ -71,66 +67,55 @@ def trans_paste(fg_img, bg_img, alpha=1.0, box=(0, 0)):
...
-                    f'**losses:** {result.get("losses")}\n'
-                    f'**total battle count:** {result.get("battleCount")}\n'
-                    f'**tree crown wins:** {result.get("threeCrownWins")}\n'
-                ),
-                inline=True,
+        info_embed = Embed(
+            title=f'Profile of {result.get("name")}',
+            description=(
+                f'**level:** {result.get("expLevel")} (xp points: {result.get("expPoints")})\n'
+                f'**trophies:** {result.get("trophies")} (best: {result.get("bestTrophies")})\n'
```
Comment (drawbu) [inline app/cogs/player.py:74]: use an interface object to avoid intensive `.get` method

File: app/bot.py [hunk 3]
```diff
@@ -2,55 +2,56 @@
...
             return
-        super().run(config.get('token'))
 
-    async def on_message(self, message):
-        """If the bot receive a message"""
-        print(f'> {message.author.name} : {message.clean_content}')
-        await self.process_commands(message)
+        super().__init__(token=config.get('token'))
+
+        for filename in os.listdir("app/cogs"):
```
Comment (drawbu) [inline app/bot.py:32]: use `glob` instead of `os.listdir`

**Review by drawbu — CHANGES_REQUESTED** [review]

**Review by drawbu — APPROVED** [review]

**Comment by Sigmanificient** [comment]
:rocket:

**Pull request merged**